package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nest-raven",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Comment:
More of a question than an issue.

Do you think if we update mayor version of `@sentry/minimal`. Should we update mayor version of `nest-raven` as well?

Or is it fine to leave it like this and npm/yarn takes care of dependencies (warns users that they need 5.x sentry or something)?

Reply:
Yes, probably we should bump major version due to breaking changes (I personally don't read output of npm install carefully each time) and use `npm deprecate` to inform user about ability to switch to Sentry@5, thanks for notice.

Should I bump version to 5.0.0 and cleanup commit history?

Reply:
Nah it's fine, I'll merge and change it later. Thanks for PR!